classes/submission/maps/Schema.php

```diff
@@ -165,4 +175,66 @@ protected function getAvailableEditorialDecisions(int $stageId, Submission $subm
 
         return $decisionTypes;
     }
+
+    /**
+     * Get issues associated with submissions. Results are keyed by submission ID.
+     *
```

Comment:
```suggestion
     * @param Enumerable<int, Issue[]> $submissions Issues associated with submissions. Keyed by submission ID. **/
     *
```

Reply:
No, not required to be keyed by submission  ID